packages/uni_ui/lib/common/generic_squircle.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:figma_squircle/figma_squircle.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class GenericSquircle extends StatelessWidget {
```

Comment:
I don't like the name `GenericSquircle`, I would prefer just `Squircle`, sorry ahah